using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -573,37 +820,38 @@ var targetParameters = new ACPTargetParameters(mboxParameters, profileParameters
 // Target API Call
 ACPTarget.prefetchContent(prefetchList, targetParameters).then(success => console.log(success)).catch(err => console.log(err));
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Clear prefetch offer cache
 
-Use this API to clear prefetched offers cached in memory.
+## resetExperience
+
+This API resets the user's experience by removing the visitor identifiers and resetting Target session. Invoking this API also removes previously set Target user ID and custom visitor IDs, Target Edge Host and session information from persistent storage.
```

Comment:
Please reword the 2nd sentence so that it reads:
"Invoking this API also removes the previously set Target user ID and custom visitor IDs, Target Edge Host, and the session information from persistent storage.